### DISCUSSION TITLE
Is HybiParser is web socket

### DISCUSSION THREAD

**Discussion by joroMaser** [jonassjoh/dspW245#13] [General]
Are the algorithm of encode and decode in HybiParser is known algorithm of WebSocket or another known protocol? 
Or maybe is costume algorithm of Dlink?

**Comment by jonassjoh** [comment]
It's a known algorithm. There is nothing special about the implementation (other than being incomplete), and it's not modified by D-Link.
Only reason that I wrote it myself is because I was trying to reverse engineer part of the mydlink android app. When it worked I didn't bother changing it... Because it worked.

**Comment by joroMaser** [comment]
Can you please tell me what is the name of algorithm so I can call to build-in function please?

Do you know build-in function in some library? 

Is that WebSocket algorithm? 

Thank you

**Comment by Garfonso** [comment]
It's just plain websocket. I used a javascript websocket library to comunicate with it and had no issues..
